src/envoy/mixer/BUILD

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Google Inc. All Rights Reserved.
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
don't change this, copyright year should be the year the work first published (so, oldest).